Source/ThirdParty/libwebrtc/Source/webrtc/common_video/h265/h265_sps_parser.cc

```diff
@@ -134,6 +141,11 @@ H265SpsParser::ParseShortTermRefPicSet(
     ref_pic_set.num_negative_pics = reader.ReadExponentialGolomb();
     // num_positive_pics: ue(v)
     ref_pic_set.num_positive_pics = reader.ReadExponentialGolomb();
+#if WEBRTC_WEBKIT_BUILD
+    if (!reader.Ok() || ref_pic_set.num_negative_pics > 10000 || ref_pic_set.num_positive_pics > 10000) {
```

Comment:
Might be best to use a constant like max_ref_value and use this value in all places. I am not able to look at the standard now but I wonder whether a smaller value would not be better. The reorder buffer size in H264 is capped to 16 for instance.

Reply:
I added these based on some private feedback from Youenn:

```
#if WEBRTC_WEBKIT_BUILD
const uint32_t kMaxSPSLongTermRefPics = 32;
const uint32_t kMaxSPSPics = 16;
const uint32_t kMaxSPSShortTermRefPics = 64;
#endif
```